teacher-profile-task

SidraShahid0510 opened the pull request:
### What does this PR do?
this PR is created to make changes in teacher-profile.html
### Description of Task to be completed?
-new branch
-changed font sizes and color
-changes padding and margin
-navbar ul position and font size changes
-bottom border on navbar

### How should this be manually tested?

### Any background context you want to provide?

SidraShahid0510 committed:
teacher-profile-task

-new branch
-changed font sizes and color
-changes padding and margin
-navbar ul position and font size changes

inastefansdottir reviewed (changes requested):
Hey Sidra,

Great job on this! The styling looks fantastic. The colours, text size, and layout are all spot on. I did notice a couple of minor details that could be tweaked:

- On line 213, the `pl-` class is applied twice, it only needs to be used once.
- The button should include the following classes for consistency with the figma design: `cursor-pointer`, `font-poppins`, and `font-semibold` instead of `font-bold`.
- Also, according to the Figma design, the date of birth should be formatted as **19/11/1988**, not **19.11.1988**

Everything else looks amazing, nice work as always!

SidraShahid0510 committed:
updated teacher-profile-task

Hello, I have done following changes.
-change font size from font-bold to font semi-bold
-cursor pointer
-removed pl
-formatted date

SidraShahid0510 commented:
Hello, I have done following changes.
-change font size from font-bold to font semi-bold
-cursor pointer
-removed pl
-formatted date

inastefansdottir reviewed (approved):
This looks perfect. I don't see any faults in the code, and the changes I requested have been implemented. I'm happy to approve this pull request.
